scripts/setup/terminate-psql-sessions

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xe
```

Comment:
This is better as:
```
#!/bin/bash
set -x
set -e
```

to avoid someone invoking it directly and losing the flags. But we do this elsewhere, so I'll fix that in a separate PR.